include/cinder/gl/Texture.h

```diff
@@ -269,7 +278,7 @@ class TextureBase {
 		bool				mImmutableStorage;
 		GLfloat				mMaxAnisotropy;
 		GLint				mInternalFormat, mDataType;
-		GLint				mDataFormat;
+		//GLint				mDataFormat;
```

Comment:
Yes, it appeared it was no longer used, but I didn't want to remove it altogether because I didn't know why it was there in the first place.